src/System.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -19,9 +19,9 @@
 namespace System.IdentityModel.Tokens
 {
     /// <summary>
-    /// Error codes and messages
+    /// Log messages and codes
```

Comment:
I opened up a beta 8 issue for this. https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/240